python/kvikio/tests/test_basic_io.py

```diff
@@ -273,3 +274,18 @@ def test_different_bounce_buffer_sizes(tmp_path, size, tasksize, buffer_size):
                     f.write(a)
                     assert f.read(b) == b.nbytes
                     cupy.testing.assert_array_equal(a, b)
+
+
+def test_bounce_buffer_clear(tmp_path):
+    """Test clearing the bounce buffer"""
+    filename = tmp_path / "test-file"
+    with kvikio.defaults.set_compat_mode(True), kvikio.defaults.set_num_threads(1):
+        with kvikio.CuFile(filename, "w") as f:
+            with kvikio.defaults.set_bounce_buffer_size(1024):
+                f.write(cupy.arange(10))  # populate the bounce buffer
+                assert kvikio.buffer.bounce_buffer_clear() == 1024
+                f.write(cupy.arange(10))  # populate the bounce buffer
+            with kvikio.defaults.set_bounce_buffer_size(2048):
+                with kvikio.CuFile(filename, "w") as f:
+                    f.write(cupy.arange(10))  # populate the bounce buffer
+        assert kvikio.buffer.bounce_buffer_clear() == 2048
```

Comment:
*question*: Is this indentation right? I thought that the context manager on line 288 would reset the bounce buffer size to the default (16MiB) size, and so this clear would return `0` (because when we exited the block we had to clear the bounce buffers).

Reply:
The bounce buffer isn't cleared on context exit, added the following comment:
```python
            # Notice, since the bounce buffer size is only checked when the buffer
            # is used, we populate the bounce buffer in between we clear it.
```